app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -13,20 +13,20 @@
 
 <p>If you’re studying for a degree and interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
 
-<p>Internships last 3 weeks, start in June and you’ll be paid £300 per week.</p>
+<p>Internships last 3 weeks, start in June or July and you’ll be paid £300 per week.</p>
 
 <section class="col col-640">
   <%= render Content::QuoteComponent.new(
     text: "The support I had was amazing, I was treated like a member of staff, and I was welcomed by everyone.",
     name: "Chloe, former teaching intern",
     background: "grey",
-    large: false
+    large: true
```

Comment:
@philspray I know the internships team asked for quotes to be more prominent but I'm not sure this is the right thing to do

The text on desktop is now very dominant and I worry it might overshadow the guidance? (On mobile, it's virtually the same size whether the flag is false or true, which is odd!)

There are other options such as background colour (we've used yellow before) but maybe one to discuss with @EllieNodder and @garethwilliamsdesign ?